test/passing/cinaps.ml

```diff
@@ -1,3 +1,7 @@
+File "cinaps.ml", line 1, characters 0-4:
+Warning 50: unattached documentation comment (ignored)
+File "cinaps.ml", line 1, characters 0-4:
+Warning 50: unattached documentation comment (ignored)
```

Comment:
What creates these warnings?

Reply:
They are generated by the ocaml parser itself. It is responsible for turning doc-comments into attributes and attaching them to the right place.

Reply:
I overlooked them but they make the test fail obviously !
This can be fixed for this test by adding a `.ref` file.

The locations are not right.

Reply:
It seems they weren't needed, I just removed them. I have no idea what happened.